conf/messages.en

```diff
@@ -475,8 +475,9 @@ charityRegistrationNumber.title = What is your charity registration number (CHRN
 charityRegistrationNumber.heading = What is your charity registration number (CHRN)?
 charityRegistrationNumber.p1 = Your charity registration number is on any mail that the charity regulator has sent you.
 charityRegistrationNumber.p2 = In Scotland, your registration number will start with SC.
```

Comment:
```suggestion
charityRegistrationNumber.p2 = This could be up to 7 characters and must begin with either 1 or 2 letters, followed by 1 to 5 numbers.
```

As per confluence - this needs updating as well

Reply:
Modified,